src/utils/i18n.js

```diff
@@ -42,7 +42,7 @@ export const resources = {
         "The longer you lock your IQ, the more hiIQ you will receive. You can also get additional hiIQ by locking more IQ or extending the lock period. hiIQ decays slowly over your locking period, eventually reaching 1-to-1 with IQ.",
       locked: "locked",
       login_info_eos: "Login on EOS Wallet to bridge your tokens",
-      login_info_eth: "Login on ETH Wallet to swap your tokens",
+      login_info_eth: "Login on ETH Wallet to lock your tokens",
```

Comment:
![image](https://user-images.githubusercontent.com/1288106/120461840-a83ecd00-c39a-11eb-8fcd-43470931378c.png)

https://github.com/EveripediaNetwork/iq-bridge-ui/issues/31

login to swap info message **in locking** should be "login to lock" <- Only in locking